test/areaOfCircleSector.test.js

```diff
@@ -0,0 +1,12 @@
+const { areaOfCircleSector } = require("../lib");
+
+describe("areaOfCircleSector", () => {
+  test("throws an error for invalid types", () => {
+    expect(() => areaOfCircleSector("pizza", 60)).toThrow("Invalid Argument");
+    expect(() => areaOfCircleSector(5, "pizza")).toThrow("Invalid Argument");
+  });
+
+  test("returns the area of the sector", () => {
+    expect(areaOfCircleSector(5, 60)).toBe(13.09);
+  });
```

Comment:
Are there lower and upper bounds to sectors?

Reply:
Good catch. Will fix.

Reply:
👍

Reply:
@ktilcu I added the requested tests. Something happened the last time I synched with upstream and commits got duplicated. Not sure what that's all about.